### PR TITLE
Remove duplicate code

### PR DESCRIPTION
**Pull request by LambertWSJ**

### CODE REVIEW

File: riscv.c [hunk 1]
```diff
@@ -1426,8 +1426,6 @@ void rv_step(struct riscv_t *rv, int32_t cycles)
             if (!op(rv, inst))
                 break;
```
Comment (jserv) [inline riscv.c:1426]: Remove this blank line as well.

Comment (LambertWSJ) [reply riscv.c:1426]: Sorry I miss this. I have updated.

File: riscv.c [hunk 2]
```diff
@@ -1435,10 +1433,9 @@ void rv_step(struct riscv_t *rv, int32_t cycles)
             TABLE_TYPE_RVC op = jump_tablec[c_index];
             assert(op);
             rv->inst_len = INST_16;
-            if (!op(rv, inst))
-                break;
-
         }
+        if (!op(rv, inst))
+            break;
```
Comment (jserv) [inline riscv.c:1439]: Add new blank line here.

**Comment by jserv** [comment]
Use `git rebase -i` to squash two commits into one.

**Comment by jserv** [comment]
Thank @LambertWSJ for contributing!

**Pull request merged**